### DISCUSSION TITLE
Export read list to an importable file

### DISCUSSION THREAD

**Discussion by lbesnard** [gotson/komga#1554] [Ideas]
### Describe your suggested feature

On the reading list page, someone could download the reading list as a JSON, XML or CBL, whatever format is the most suitable. 

This way, it would be possible to export and re-import reading lists across various komga instances. In my case I have a few instances of komga, and I would just like to be able to re-import a list created on a different instance. But we could think of it as a great way to share reading lists with other users

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
duplicate of #770

**Comment by lbesnard** [comment]
sorry for that. I searched with the wrong key words obviously

**Comment by gotson** [comment]
No worries, it's mostly for my own record, so I can check discussions on previously closed / stale issues too.

**Comment by kikilewiwi** [comment]
Would be great to have the same for collections. Be able to export/import them.

**Comment by Cirsten** [comment]
I would like this as well. I know you wrote that it can be done via API (in the closed #770). But that's beyond me. An in program solution would be much preferred.

Unless someone has created a script I could use??

**Comment by Budlyte** [comment]
How are comic IDs generated? Unless the system to import has generated the same IDs for its issues as the exporting system then what currently exists wouldn't work well.

Example output from API query for readlists.  Comment won't keep my formatting when I post, so leaving a screenshot.
![image](https://github.com/gotson/komga/assets/4451365/70460034-faff-42e2-bfbf-2dc9fb62c73f)


I believe ISBN would be needed as part of the export/import. These can be difficult to acquire since it looks like CV still doesn't include it in their API, so Mylar doesn't get it, and many digital covers don't include it either.
